vrecord

```diff
@@ -1184,67 +1194,74 @@ else
     middleoptions+=(-color_range mpeg)
 fi
 
-if [[ "${qctoolsxml_choice}" = "Yes" ]] ; then
-    if [[ ! "$(which qcli)" ]] ; then
-        _report -w "Please install qcli to use the qctools reporting option."
-        _report -w "Such as \`brew install qcli\`."
-        exit 1
-    fi
+if [[ "${media_player_choice}" = "ffplay" ]] ; then
     "${FFMPEG_DECKLINK[@]}" \
-        2> >(tee "${logdir}/${id}${capturelogsuffix}" /tmp/vrecord_input.log >/dev/null) \
```

Comment:
This chunk of changes is only based on indentation so I'd suggest to undo it to reduce changes.